README.rst

```diff
@@ -150,6 +150,20 @@ Tune security settings (these are default):
           # Set X-Frame-Options
           frame_options: sameorigin
 
+Tune the CustomLog format:
+
+.. code-block:: yaml
+
+    parameters:
+      apache:
+        server:
+          site:
+            foo:
+              enabled: true
+              type: static
+              custom_log_format: >-
```

Comment:
I thought about this but was too lazy to go through it ;-)
I'll amend the PR based on your comment.